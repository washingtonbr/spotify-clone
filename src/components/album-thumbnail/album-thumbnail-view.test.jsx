import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import AlbumThumbnailView from './album-thumbnail-view';

describe('AlbumThumbnailView', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create((
        <MemoryRouter>
          <AlbumThumbnailView
            id="XPTO"
            cover="XPTO"
            name="XPTO"
            artists={['XPTO']}
          />
        </MemoryRouter>
      ))
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
