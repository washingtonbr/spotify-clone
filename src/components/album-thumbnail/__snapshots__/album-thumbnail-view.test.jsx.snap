// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlbumThumbnailView renders correctly 1`] = `
<article
  className="sc-AxjAm duEsRS"
>
  <a
    href="/album/XPTO"
    onClick={[Function]}
  >
    <img
      alt="XPTO"
      className="sc-AxirZ cTKAUo"
      src="XPTO"
    />
    <h2
      className="sc-AxiKw dfKjlj"
    >
      XPTO
    </h2>
    <h3
      className="sc-AxhCb dxgIcC"
    >
      XPTO
    </h3>
  </a>
</article>
`;
